app/Http/Controllers/Api/Address/AddressController.php

```diff
@@ -79,6 +86,12 @@ public function update(Request $request, Address $address)
         }
 
         $address->fill($this->arrayKeysToSnakeCase($data));
+
+        $contact = Contact::find($address->contact_id);
+        if($contact){
+            $contactAddresAllowed = $this->checkDoubleAddressAllowed($contact, $address);
```

Comment:
`contactAddresAllowed` address is met dubbel s aan het eind

Reply:
Variable name changed (address with double s)